doc/community/extensions.rst

```diff
@@ -92,6 +92,14 @@ Emacs Lisp
 
 .. _Cask: https://github.com/cask/cask
 
+Julia
+-----
+
+* :jl:`gdkrmr/flycheck-julia` makes linting for Julia_ available via Lint.jl_.
```

Comment:
no idea what this means :D I supposed that this was some kind of tag, I will change it to `:gh:` then.